formidling/src/main/java/no/nav/ung/sak/formidling/template/dto/TemplateInnholdDto.java

```diff
@@ -0,0 +1,7 @@
+package no.nav.ung.sak.formidling.template.dto;
+
+/**
+ * Marker for template innmat dto'er
+ */
+public sealed interface TemplateInnholdDto permits InnvilgelseDto {
```

Comment:
Kvifor bruker du sealed interface her og ikkje sealed class? Basert på navnet så ville eg tenkt at dette er ei klasse.

Reply:
Fordi denne interface'en skal bare arves av forhåndsbestemte klasser, og den trenger ikke å være en klasse fordi det er en marker og har ingenting som skal arves av implementasjonene. Det er for å tydelig markere Dto'er som er koblet til enkeltbrev. Etterhvert vil det komme OpphørDto, AvslagDto, EndringDto.  

Det er tilsvarende BekreftetAksjonspunktDto (som er abstrakt klasse da)